screens/Registration/InformationScreen/InformationScreen.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { View, TextInput, TouchableOpacity, Text, Image, SafeAreaView } from 'react-native';
+import Styles from './styles';
+const Form: React.FC = () => {
+  // State for form fields can be added here
+
+  return (
+
+    <SafeAreaView style={Styles.container}>
+   
+      <View style = {Styles.icon_container}>
+        <Image source ={require("../../../assets/icon.png")} style={Styles.image} />  
+      </View>
```

Comment:
Same here, icon as an image is used. Should refactor. Then, you won't need to wrap it in a `View` anymore.